modules/csg/doc_classes/CSGShape3D.xml

```diff
@@ -89,7 +89,7 @@
 		<member name="operation" type="int" setter="set_operation" getter="get_operation" enum="CSGShape3D.Operation" default="0">
 			The operation that is performed on this shape. This is ignored for the first CSG child node as the operation is between this node and the previous child of this nodes parent.
 		</member>
-		<member name="snap" type="float" setter="set_snap" getter="get_snap" default="0.001">
+		<member name="snap" type="float" setter="set_snap" getter="get_snap" default="0.001" deprecated="Snap is not used by CSG anymore.">
 			Snap makes the mesh vertices snap to a given distance so that the faces of two meshes can be perfectly aligned. A lower value results in greater precision but may be harder to adjust. The top-level CSG shape's snap value is used for the entire CSG tree.
```

Comment:
If `snap` really does nothing now this description is lying to you.

Reply:
Will update

Reply:
```
<member name="snap" type="float" setter="set_snap" getter="get_snap" default="0.001" deprecated="Snap is not used by CSG anymore.">
	CSG no longer uses snap and has been removed because snapping is not implemented in the backing CSG manifold library.
</member>
```